docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -22,7 +22,10 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # How to: Verify that Strings Are in Valid Email Format
-The following example uses a regular expression to verify that a string is in valid email format.  
+The following example is more complex than previous examples and uses a regular expression to verify that a string is in valid email format.  
```

Comment:
This change is problematic. Previous here has no referent. The implicit referent is the TOC, but an extremely small number of people navigate here from the TOC or use the TOC. There's also no clear relationship in terms of complexity between examples. What is the case is that this example is complex. It's OK to phrase this as "The following example uses a complex regular expression..."